package.json

```diff
@@ -13,23 +13,26 @@
   "unpkg": "dist/manifold-plan-table/manifold-plan-table.js",
   "license": "MIT",
   "scripts": {
+    "prebuild": "npm run version-from-git",
     "build": "stencil build",
+    "postbuild": "npm run format:changelog",
```

Comment:
I keep going back-and-forth about the order of these events:

1. Update `package.json` with version from Git tag
2. Build Stencil for npm
3. Format the CHANGELOG (and also add the commit comparisons)

#1 **MUST** happen before #2 (the `package.json` version gets shipped with the build when the components submit metrics), but #3 could happen at any point. My thinking is that the build is more important, but overall I’m 🍌.

Reply:
And I do like formatting the changelog regularly—on every build—because it’s quick and if everyone does it while they’re working, it’s more likely to work when it comes time for publish.